Ground_oM/ITestProperties/SPTTestProperties.cs

```diff
@@ -0,0 +1,71 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2025, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+
+using System;
+using System.ComponentModel;
+using System.Collections.Generic;
+using BH.oM.Base;
+using BH.oM.Quantities.Attributes;
+
+namespace BH.oM.Ground
+{
+
+    [Description("Properties related to the SPT Test.")]
+    public class SPTTestProperties : BHoMObject, ITestProperties
+    {
+        /***************************************************/
+        /**** Properties                                ****/
+        /***************************************************/
+
+        [Description("SPT Hammer Number (ISPT_HAM).")]
+        public virtual string SPTHammerNumber { get; set; }
+
+        [Description("Type of SPT Test (SPT_TYPE).")]
+        public virtual string SPTTestType { get; set; }
+
+        [Length]
+        [Description("Depth to water at time of test (ISPT_WAT).")]
```

Comment:
```suggestion
        [Description("Depth to water at time of test, set to NaN if dry (ISPT_WAT).")]
```
Just need to capture this when reading an AGS file.